### ISSUE TITLE
Decoding support for Spanish and Russian?

### ISSUE THREAD

**Issue by ZoruaLoL** [piratesephiroth/LaMulanaScript#3]
I tried to use the program on the Spanish localization files, but all I got was gibberish, at closer inspection I noticed it (and the Russian localization as well) use unique, much shorter font00.png files compared to the EN/JP scripts

**Spanish**
![font00](https://github.com/user-attachments/assets/49cc9525-2761-46ee-9886-7a08885c86ed)

**Russian**
![font00](https://github.com/user-attachments/assets/c120f57c-4576-4ee4-9e8f-ead06e186c26)

**English/Japanese**
![font00](https://github.com/user-attachments/assets/73fe7e6d-ea1d-4b0e-8f56-82318032b3ac)

I tried to rearrange the characters in the fontChars.txt but ran into several problems since accented vowels don't appear to have full-width variants and even without them I constantly got an error message telling me there were "duplicate characters" on it...

**Comment by piratesephiroth** [comment]
yeah the difficult part is to create a proper character list that matches the glyphs in the image.
